app/components/swapper.tsx

```diff
@@ -69,11 +70,28 @@ const Swapper = () => {
     }, 2000);
   };
 
-  const handleTokenSwap = () => {
+
+const STABLE_TOKENS: Token[] = ['USDT', 'USDC'];
+
+const handleTokenSwap = () => {
+  // ensures at least one of the tokens is a stable coin
+  const isValidSwap = 
+    STABLE_TOKENS.includes(fromToken) || 
+    STABLE_TOKENS.includes(toToken);
+
```

Comment:
This is not the intended functionality you were to implement, you were supposed to make sure that when the swap button is clicked, an unstable token doesn't end up in the stable token's placeholder.